crates/bevy_pbr/src/light.rs

```diff
@@ -768,7 +765,10 @@ pub(crate) fn assign_lights_to_clusters(
             ClusterFarZMode::Constant(far) => far,
         };
         let first_slice_depth = match (is_orthographic, requested_cluster_dimensions.z) {
-            (true, _) => camera.near,
+            (true, _) => {
+                (camera.projection_matrix.w_axis.z - 1.0)
+                    / (camera.projection_matrix.z_axis.z * 2.0)
```

Comment:
Where does this come from? Based on https://github.com/bitshifter/glam-rs/blob/main/src/core/traits/projection.rs#L147-L163 which is what we are using for our orthographic projection, it looks incorrect. Also, we swap the view space near and far to give us a reverse z projection, consistent with the reverse z perspective projection that we use. As such, I think the following would be correct, but it needs testing!
```suggestion
                // NOTE: Based on glam's Mat4::orthographic_rh(), as used to calculate the orthographic projection
                // matrix, we can calculate the projection's view-space near plane as follows:
                // component 3,2 = r * near and 2,2 = r where r = 1.0 / (near - far)
                // There is a caveat here that when calculating the projection matrix, near and far were swapped to give
                // reversed z, consistent with the perspective projection. So,
                // 3,2 = r * far and 2,2 = r where r = 1.0 / (far - near)
                // rearranging r = 1.0 / (far - near), r * (far - near) = 1.0, r * far - 1.0 = r * near, near = (r * far - 1.0) / r
                // = (3,2 - 1.0) / 2,2
                (camera.projection_matrix.w_axis.z - 1.0) / camera.projection_matrix.z_axis.z
```

Reply:
Thank you for pointing it out. A the time I couldn't find glam's version of the orthographic projection so I tried to reverse engineer it from the OpenGL version. I tested it with your solution and it seems to be working correctly (except for the floating point inaccuracies).

Reply:
As an alternative, we could just add an optional query for the OrthographicProjection component and read the plane directly from there. Less generic / wouldn't handle custom projections properly, but more exact. However in practice i dont think floating point inaccuracy really matters here.

Reply:
I think this is a good call but out of the scope of this PR.

Reply:
it would be nice for this to be a method somewhere... i guess the right place would be in the glam Projection trait.